8. Exception handling.js

```diff
@@ -1,16 +1,15 @@
 try {
-    console.log(a);
-    let a = 3;
-  } catch (err) {
-    alert('let перед использованием нужно объявить!');
-  };
-
+  console.log(a);
+  let a = 3;
+} catch (err) {
+  alert("let перед использованием нужно объявить!");
+}
 
 try {
-   let n = 1/0;
-   if (n === Infinity) {
-        throw err;
+  let n = 1 / 0;
+  if (n === Infinity) {
+    throw err;
   }
-} catch (err){
-    alert('на ноль делить нельзя!')
-};
\ No newline at end of file
+} catch (err) {
+  alert("на ноль делить нельзя!");
```

Comment:
No prompts or alerts in your code. Please use direct pass value in your code and logging to console.